### DISCUSSION TITLE
Laragon self termination

### DISCUSSION THREAD

**Discussion by kibrahim2780** [leokhoa/laragon#655] [Q&A]
Laragon does not self terminate mysql port 3306 when i restart or start my computer again after shutting down it gives error that roor user access is denied and i have to kill the port by using commands on cmd why does not it not terminate the port itself

**Comment by Pen-y-Fan** [comment]
Do you have another instance of MySQL?

Check the netstat report in Laragon menu > tools > netstat

If you have another MySQL server running (e.g. XAMPP) configure it to not start at startup.

**Comment by kibrahim2780** [comment]
yes i have checked it and mysql workbench server was also running on the
same port as laragon mysql so i changed the port and its working fine
